src/OrchardCore.Modules/OrchardCore.Flows/Controllers/AdminController.cs

```diff
@@ -83,7 +83,7 @@ public async Task<IActionResult> BuildEditor(string id, string prefix, string pr
                 CanDelete: true,
                 //Input hidden
                 //Prefixes
-                HtmlFieldPrefix: prefix,
```

Comment:
Removing this concerns me.
Have you checked that recursive parts will still work after this change? Meaning a bag part inside a flow part, something that has multiple levels of "dynamic" injection of an edit template.

Reply:
Yes, not all use cases but I tried nested things.

The `HtmlFieldPrefix` of the `ContentCard` was never used, only the `PrefixValue`.

Now we include the `HtmlFieldPrefix` (retrieved from the `ViewData.TemplateInfo`) in the `PrefixValue` on which we add an index value.

We also pass this `HtmlFieldPrefix` to the script through an html attribute so that, when we hit this controller through an ajax call, it is also included in the prefix parameter that we pass to the controller, this to set again here the `PrefixValue` of the `ContentCard`.